spec/datetimeformat.html

```diff
@@ -1266,7 +1266,7 @@ <h1>
         1. If IsTimeZoneOffsetString(_timeZoneIdentifier_) is *true*, then
           1. Let _offsetNs_ be ParseTimeZoneOffsetString(_timeZoneIdentifier_).
         1. Else,
-          1. Assert: IsValidTimeZoneName(_timeZoneIdentifier_) is *true*.
+          1. Assert: GetAvailableNamedTimeZoneIdentifier(_timeZoneIdentifier_) is not ~empty~.
           1. Let _offsetNs_ be GetNamedTimeZoneOffsetNanoseconds(_timeZoneIdentifier_, _epochNs_).
         1. Let _tz_ be ℝ(_epochNs_) + _offsetNs_.
         1. If _calendar_ is *"gregory"*, then
```

Comment:
I think we can remove the note a few lines below here that says

> A conforming implementation must recognize *"UTC"* and all Zone and Link names from the IANA Time Zone Database (and <strong>only</strong> such names), and use best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations.

It's now covered in "Use of the IANA Time Zone Database".